### ISSUE TITLE
I can't comment anymore in issue #10.

### ISSUE THREAD

**Issue by Ravizz** [yourduskquibbles/webannoyances#11]
Hey, I have a problem with commenting at your repo, in this issue: https://github.com/yourduskquibbles/webannoyances/issues/10 I've wanted to leave another reply but I simply can't, after clicking "Comment" my comment just simply doesn't show. From my experience I know that happens if blocked/banned by admin of repo or by author of issue (@anewuser in this case) , and sometimes it might be caused by adblocker, but even with adblocker  disabled, I still can't, I've no problem with commenting at other repos, also I see I can create a new issue and comment here, so I guess it haven't been you, what's the problem then, have I been blocked/banned by the author of the issue? Regardless of the answer,  I'll leave my reply here:

Just for clarity:

> @anewuser : He has edited the post I replied to here.

I've edited my comment before you've replied, and not after.

> @anewuser : calling me names

If by that you're trying to exhibit that I've been insulting you the same way like you've been insulting me on @gorhill's repo, you will fail:
- I've used a single word: a troll.
- I haven't been calling you names, I've called you by name, and that's a difference, you together with your "friend" @Hrxn, have been calling me names (insulting) for no reason, so I've just described you as a troll, stated a fact, called by name, and that's not the same, in no way I've been insulting you, don't compare me to yourself, coz I'm not at your level.

**Comment by yourduskquibbles** [comment]
You likely couldn't comment because the issue was closed by the author.

**Comment by Ravizz** [comment]
Not possible, simple closing the issue by the auhtor doesn't prevent from commenting in the issue.

**Comment by yourduskquibbles** [comment]
I don't know why then - I didn't do anything to restrict your comments and haven't changed the site settings for commenting on this repo from github defaults.

**Comment by Ravizz** [comment]
OK I've just checked and I can comment in other issues like: https://github.com/yourduskquibbles/webannoyances/issues/9#issuecomment-364224954 and https://github.com/yourduskquibbles/webannoyances/issues/6#issuecomment-364224403

**Comment by elypter** [comment]
i suspect some secret github algorithm that felt offended.

**Comment by Ravizz** [comment]
from https://github.com/contact

>  Need help with the yourduskquibbles/webannoyances project?
> You should get in contact with @yourduskquibbles. For example, you could open an issue on the repository. 

> GitHub Support is unable to help with issues specific to [yourduskquibbles/webannoyances](https://github.com/yourduskquibbles/webannoyances).
> Get in contact with @yourduskquibbles or check the project's README file.


![github2](https://user-images.githubusercontent.com/35421792/36016971-ecdfdbfe-0d75-11e8-967b-289248e8b505.png)

?

**Comment by Ravizz** [comment]
@yourduskquibbles and @elypter what do you think ?

**Comment by elypter** [comment]
it would be a serious security concern if it worked that way. i still assume that it was an automated algorithm based on reports and stress level detection algorithms. the reporting system on big sites mostly works by counting how often someone reported you and if it is over a certain threshold you get blocked or rated to some degree. moderators only act if you oppose or if it gets publicity.

**Comment by Ravizz** [comment]
It all looks weird, so you're:
- suggesting that I've been blocked not by user, but by some unknown misterious github's algorithm, but from the github's reply it's clear that they've said it was an user's action. Could you please give me a link to any kind of description/mention/comments/people talking about this algorithm?
- also you're suggesting that this algorithm was triggered because I've been reported: but I didn't do anything bad I could be reported for, it was the other people who were harassing/attacking/insulting me, and also I even never reported anybody, so it looks like: people were harassing me and i did not report them and I was not harassing anybody but they reported me? Such behaviour would be just ridiculous and unfair.

Look, I will contact github again and ask them about this algorithm.

**Comment by Ravizz** [comment]
OK, they've replied again: 

> we can confirm that this does not appear to be the result of a technical issue on GitHub.

So simply that's not any github's algorithm @elypter has been talking about.
The only physical possibilities are that I've been blocked by @anewuser either @yourduskquibbles.

Now I'm a bit confused:
- on the one hand @yourduskquibbles said that he hasn't blocked me
- on the other hand when I've asked:  

>  There has only 1 possibility left - I've been blocked by: @anewuser but I'm not sure if that is even possible to block another user from own issue while not in own repo? 

@elypter has replied: 

> it would be a serious security concern if it worked that way.

But based on the facts that he was mistaken once (with the github's algorithm), he could have made a mistake again as well.

Could you guys now simply ask @anewuser if he has blocked me?

**Comment by Ravizz** [comment]
OK I've got an answer from GitHub and already know that it was @anewuser who has blocked me (he has added me to his blacklist).

And the other (strange?) thing is that GitHub restricts a blocked user from commenting in the issue if the person who has blocked is an author of the issue. 

The another (strange?) thing is I've not found that info in GitHub's public documentation, so I have had to ask GitHub via Contact Form.

So my predictions have turned out to be right:

> But based on the facts that he was mistaken once (with the github's algorithm), he could have made a mistake again as well. 

The guy has been wrong two times.

So my question:

> Could you guys now simply ask @anewuser if he has blocked me?

is no longer valid, although it is sad that @yourduskquibbles had time to label, but not to answer my question? Anyway, let's try to ask another questions (will he answer this time?): as an admin of the repo are you able to remove from me the commenting restriction imposed by @anewuser in this issue: https://github.com/yourduskquibbles/webannoyances/issues/10 ? Because it's a bit strange, that I can't participate in **public** discussion, does the restriction mean that https://github.com/yourduskquibbles/webannoyances/issues/10 is @anewuser's a private issue? Just a rhetorical question coz it is known that it is not. And what do you thing about github's possibilty to restrict public issues from commenting by private users blacklists? That's looks kinda weird doesn't it?

**Comment by yourduskquibbles** [comment]
> As an admin of the repo are you able to remove from me the commenting restriction imposed by @anewuser in this issue: #10 ?

Unfortunately I don't see a way to do that. Although admittedly I am not an expert in every feature of github.

> I want to, but I'm unable...what do I do now?

I am reading here too :-)

**Comment by Ravizz** [comment]
Well, nothing more to add than what is already in: https://github.com/yourduskquibbles/webannoyances/issues/10#issuecomment-365919084, the wiki demonstration seems to be OK, although maybe to put in one of the slideshows a footer example as an opposite to the header one.
